rst_files/testing.rst

```diff
@@ -457,20 +461,24 @@ The basic object is the macro ``@test``
 
 Tests will pass if the condition is ``true``, or fail otherwise
 
-If a test is failing, we should *flag it and move on*
+If a test is failing, we should flag it and continue to run our tests
```

Comment:
Wasn't exactly sure what we were getting at here and in the next sentence - would someone normally just delete a test that wasn't working? The idea is we keep tests that are failing and don't stop running through the tests?

Reply:
I think a lot of beginning programmers will just comment out/delete failing tests, because they want the build to proceed. The idea here is that they should use `@test_broken` to mark it instead. Will make that more clear.

Reply:
Done.